Documentation/PageTsconfig/Mod/WebInfo.rst

```diff
@@ -74,41 +74,24 @@ Example: Override the field definitions in the info module
 menu.function
 =============
 
-..  confval:: menu.function
-    :name: mod-web-info-menu-function
-    :type: array
-
-    Disable elements of the "Function selector" in the document header of the module. The keys for single
-    items can be found by browsing *System > Configuration > $GLOBALS['TBE_MODULES_EXT']*.
+..  versionchanged:: 12.0
+    The TSconfig option :tsconfig:`mod.web_info.menu.function` has been removed
+    with TYPO3 v12.0.
 
-    ..  figure:: /Images/ManualScreenshots/Info/FunctionMenuInfoModule.png
-        :alt: The function menu of the Info module
+..  _pageblindingfunctionmenuoptions-webinfo-migration:
 
-        The function menu of the Info module
+Migration from menu.function to options.hideModules
+---------------------------------------------------
 
-    ..  warning::
+..  todo: link to options.hideModules once it is documented
```

Comment:
See here: https://docs.typo3.org/m/typo3/reference-tsconfig/12.4/en-us/UserTsconfig/Options.html#hidemodules

Reply:
Yes, it is quite counter intuitive as it seems like a page tsconfig option was superseeded by one in user tsconfig. We had some discutssions here: https://forge.typo3.org/issues/103627 and here: https://forge.typo3.org/issues/103627

Reply:
ok